### PR TITLE
Updated headlines and corrected grammatical errors

### LINKED ISSUE #25: Grammatical Errors
I saw some grammatical errors in the website which might leave bad impression on potential client or employer, i wish to work on that and improvise it with more effective tag lines, Assign this to me under hacktoberfest

### PR DESCRIPTION
**Pull request by bilaljaved01**
Fixes #25

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,10 +39,9 @@
             <p class="title">-INTRODUCTION</p>
             <p class="hi">Hi There!👋</p>
             <h1 class="my-name">I'm <span>Divya Kumari</span></h1>
-            <p class="slogun">I help startup to create stunning websites,
-              that make impact on market.</p>
+            <p class="slogun">I empower startups to craft compelling websites that leave a lasting impression on the market.</p>
```
Comment (Iamdivyak) [inline index.html]: that's not a grammar mistake. you changed words. I would like to keep the previous one.

**Review by Iamdivyak — COMMENTED** [review]
change the slogan as it was before.

File: index.html (hunk 1 above)
Comment (bilaljaved01) [reply index.html]: The grammatical error in the sentence is the incorrect placement of "to." It should be "I help startups create stunning websites that make an impact on the market." , but sure i will leave it as is

**Comment by bilaljaved01** [comment]
Can you give it label of hacktoberfest-accepted, as it shows "not - accepted" on hacktoberfest website

**Comment by Iamdivyak** [comment]
Thank you, @bilaljaved01, for contributing. and bearing my demands😅

**Comment by bilaljaved01** [comment]
it's okay, hope to collaborate again in future! Have a good one

**Pull request merged**